Description: Fix small typo in manpage for esmtprc.5 manpage
 reported by Jakub Wilk.
Origin: vendor
Bug-Debian: https://bugs.debian.org/528343
Author: Salvatore Bonaccorso <salvatore.bonaccorso@gmail.com>

--- a/esmtprc.5
+++ b/esmtprc.5
@@ -72,7 +72,7 @@
 \fBhelo\fR
 Set the hostname to identify as when sending HELO or EHLO commands.
 (This is a per identity option, as it should be the name you are
-seen as from the connected host, which may very with host to host
+seen as from the connected host, which may vary with host to host
 due to NAT or different naming schemes).
 
 .TP
